packages/0xcert-scaffold/src/core/asset-ledger.ts

```diff
@@ -266,7 +273,14 @@ export interface AssetLedgerInfo {
    * Example: https://example.com/id/
    * Asset 1 URI will become: https://example.com/id/1
    */
-  uriBase: string;
+  uriPrefix: string;
+
+  /**
+   * Uri postfix for metadata URI-s. After uriPrefix and assetId postfix is automatically appended for each asset.
```

Comment:
* URI postfix for metadata URIs. After uriPrefix and assetId, postfix is automatically appended for each asset.